runtime-infra/pn-monitoring.yaml

```diff
@@ -93,6 +93,16 @@ Parameters:
     Default: 1
     Description: 'Batch size for the diagnostic logger lambda'
 
+  SSLCertExpirationCheckLambdaName:
+    Type: String
+    Default: 'pn-SSLCertExpirationCheck'
```

Comment:
@ViVeniero perché ti porti il nome della lambda in input?

Reply:
@ViVeniero ora che definirai il log group dentro il frammento, potrai usare il nome della risorsa per non duplicare il nome della lambda dappertutto.